integration-test/sdkContracts/contracts/src/main/resources/META-INF/services/io.provenance.scope.contract.contracts.ContractHash

```diff
@@ -0,0 +1 @@
+io.p8e.contracts.testframework.ContractHash1630609561039
```

Comment:
you can add the following to `integration-tests/.gitignore`. This will ignore all the bootstrapped generated files.

```
ContractHash*.kt
ProtoHash*.kt
io.provenance.scope.contract.contracts.ContractHash
io.provenance.scope.contract.proto.ProtoHash
```